google/resource_container_cluster_test.go

```diff
@@ -2229,3 +2250,18 @@ resource "google_container_cluster" "shared_vpc_cluster" {
 	]
 }`, projectName, org, billingId, projectName, org, billingId, acctest.RandString(10), acctest.RandString(10), name)
 }
+
+func testAccContainerCluster_withResourceLabels() string {
+	testId := acctest.RandString(10)
```

Comment:
Ideally, usually we pass this kind of thing in as an argument, so we have control over whether configs are identical or different when we're referencing them.

Reply:
done